### ISSUE TITLE
Invalid "type" in esm/package.json

### ISSUE THREAD

**Issue by a2** [wokwi/rp2040js#89]
I'm seeing the below error from [_esbuild_](https://esbuild.github.io) when I build a project that includes _rp2040js_. If I change the value there (locally) to "module" then everything still works (from my testing).

```
▲ [WARNING] "esm" is not a valid value for the "type" field

    node_modules/rp2040js/dist/esm/package.json:1:8:
      1 │ {"type":"esm"}
        ╵         ~~~~~

  The "type" field must be set to either "commonjs" or "module".
```

**Comment by urish** [comment]
Thanks for reporting! TBH I don't really remember why we used "esm", but let's fix this

**Comment by urish** [comment]
Fix released as 0.14.7
